testInput2.txt

```diff
@@ -0,0 +1,4 @@
+gps_timestamp lat lng alt roll_deg pitch_deg yaw_deg
+1140885738.407125 48.129872 11.582905 10.015034 -177.971817 -3.228588 10.700191
```

Comment:
Actually, you don't need this file, you can generate a temp file used to test. That is convenient for you to test the field value and record size.